icu4c/source/test/intltest/restsnew.cpp

```diff
@@ -1198,5 +1200,66 @@ NewResourceBundleTest::TestGetByFallback() {
     status = U_ZERO_ERROR;
 
 }
+
+
+#define REQUIRE_SUCCESS(status) { \
+    if (status.errIfFailureAndReset("line %d", __LINE__)) { \
+        return; \
+    } \
+}
+
+#define REQUIRE_ERROR(expected, status) { \
+    if (status.expectErrorAndReset(expected, "line %d", __LINE__)) { \
+        return; \
+    } \
+}
+
+void
+NewResourceBundleTest::TestFilter() {
```

Comment:
Please add a comment describing what functionality this is trying to test.

Reply:
Done